docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-server.adoc

```diff
@@ -57,3 +57,19 @@ WARNING: Be careful, because changing the runtime may result in unusable state o
   because of incompatible configuration.
 +
 image::eclipse-plugin/payara-server/reconfig-runtime.png[Runtime Configuration]
+
+[[auto-publish-hot-deploy]]
+== Auto Publish & Hot Deploy
+
+The Auto Deploy and Hot Deploy mode are helpful for developers to run and test 
+an application immediately after making changes to its sources without restarting 
+the Server or manual redeployment to maximize your productivity.
+
+To enable the automatic publishing from the Eclipse IDE, open The 
+*Server Overview* panel and expand the *Auto Publish* section to reconfigure the setting.
```

Comment:
```suggestion
*Server* panel, double click the server name and expand the *Publishing* section to reconfigure the setting.
```
Being new to Eclipse I found this hard to follow, not knowing you had to double click 'Payara Server 5' or whatever the server was called. The options were 'Servers' and 'Publishing' not 'Server Overview' and 'Auto Publish' in Eclipse 2021-09